release-notes/logstash-output-opensearch-release-notes-2.0.1.md

```diff
@@ -0,0 +1,12 @@
+## Version 2.0.1 Release Notes
+Compatible with OpenSearch 1.3+, 2.1.0
```

Comment:
Should this be `2.0` or `2.1.0`?

Reply:
It's also compatible with `2.1.0` so kept it at that.

Reply:
Maybe `1.3+, 2.0+` so that users know they can use it with a 2.0.0 cluster.